webapp/src/cloud_pricing/modals.tsx

```diff
@@ -0,0 +1,172 @@
+        </SizedGenericModal>
+    );
+};
+
+export const CloudFreeTrialModalAdmin = (modalProps: Props) => {
+    const {formatMessage} = useIntl();
+    const dispatch = useDispatch();
+
+    const trialTaken = useSelector(isCloudTrialCompleted);
+
+    // Note: This modal should never be triggered if currently in trial--the calls button will not be feature restricted.
+    // So the default is trialNotTaken:
+    let headerText = formatMessage({defaultMessage: 'Try channel calls with a free trial'});
+    let confirmButtonText = formatMessage({defaultMessage: 'Try free for 30 days'});
+    let bodyText = formatMessage({defaultMessage: 'Channel calls give you a quick, audio-first way of interacting with your colleagues. Get the full calls experience when you start a free, 30-day trial.'});
+    let confirmThunk: Thunk = requestTrial();
+
+    if (trialTaken) {
+        headerText = formatMessage({defaultMessage: 'Upgrade to use calls in Channels'});
+        confirmButtonText = formatMessage({defaultMessage: 'View plans'});
+        bodyText = formatMessage({defaultMessage: 'Channel calls give you a quick, audio-first way of interacting with your colleagues. Upgrade to the Professional Plan to use calls in Channels and Group Messages.'});
+        confirmThunk = displayCloudPricing();
+    }
+
+    return (
+        <SizedGenericModal
+            id={IDAdmin}
+            {...modalProps}
+            modalHeaderText={headerText}
+            confirmButtonText={confirmButtonText}
+            cancelButtonText={formatMessage({defaultMessage: 'Back'})}
+            handleConfirm={() => dispatch(confirmThunk)}
+            showCancel={true}
+            onHide={() => null}
+            components={{FooterContainer}}
+        >
+            <VerticalSpacer size={22}/>
+            {bodyText}
+        </SizedGenericModal>
+    );
+};
+
+export const CloudFreeTrialSuccessModal = (modalProps: Props) => {
+    const {formatMessage} = useIntl();
+
+    return (
+        <SizedGenericModal
+            id={IDSuccess}
+            {...modalProps}
+            confirmButtonText={formatMessage({defaultMessage: 'Done'})}
+            handleConfirm={() => null}
+            onHide={() => null}
+        >
+            <div>
+                <Centered>
+                    <SuccessImage/>
+                </Centered>
+                <Heading>
+                    {formatMessage({defaultMessage: 'Your trial has started!'})}
+                    <p>{formatMessage({defaultMessage: 'Explore the benefits of Enterprise'})}</p>
+                </Heading>
+                {formatMessage({defaultMessage: 'Welcome to your Mattermost Enterprise trial! It expires on August 1st, 2021. You now have access to guest accounts, automated compliance reports, and mobile secure-ID push notifications, among many other features. View all features on our documentation.'})}
```

Comment:
```suggestion
                {formatMessage({defaultMessage: 'Welcome to your Mattermost Enterprise trial! It expires on August 1st, 2021. You now have access to guest accounts, automated compliance reports, and mobile secure-ID push notifications, among many other features. [View a list of features on our pricing page](https://mattermost.com/pricing/). Ready to dive in? [Visit our documentation to get started:](https://docs.mattermost.com/overview/index.html).'})}
```

@cpoile: I've hit a blank for how to format the links - hopefully I got it right, but apologies if I didn't.

Reply:
@justinegeffen Yep it's good -- I made the minor changes needed, thanks!